public/images/logo-small.svg

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These weren't used, so get 'em outta here! I suppose it's not terrible to use github for backing up files, but it's not typical to use it for that.